src/server.ts

```diff
@@ -1,15 +1,26 @@
 import { Pool } from 'pg'
 import * as schedule from 'node-schedule'
-import * as Redis from 'ioredis'
+import Redis from 'ioredis'
 import { FastifyInstance } from 'fastify'
 import { PluginsServer, PluginsServerConfig } from './types'
-import { version } from '../package.json'
-import { setupPlugins } from './plugins'
-import { startWorker } from './worker'
+import { startQueue } from './worker/queue'
 import { startFastifyInstance, stopFastifyInstance } from './web/server'
 import { Worker } from 'celery/worker'
+import { version } from '../package.json'
+import { PluginEvent } from 'posthog-plugins'
+import Piscina from 'piscina'
+
+function overrideWithEnv(config: PluginsServerConfig): PluginsServerConfig {
+    const newConfig: Record<string, any> = { ...config }
+    for (const [key, value] of Object.entries(config)) {
+        if (process.env[key]) {
+            newConfig[key] = process.env[key]
+        }
+    }
+    return config
+}
```

Comment:
:+1: As as side note, I think we could move to only env var-based server configuration instead of passing in JSON (another PR).

Reply:
I tend to agree! We might want a CLI override (e.g. `--database-url=has://priority/over/env`), yet agreed that the current `--config={json}` approach is rather raw.